sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2021-11-09)
```

Comment:
Should this be 1.0.0 ?

Reply:
This one package is being released in beta. It is the first iteration of this package.

Reply:
All right!